app/Models/Member.php

```diff
@@ -341,4 +342,21 @@ public function botResponse(): array
                 . implode(PHP_EOL, $properties),
         ];
     }
+
+    /**
+     * Returns the member's forum information via the bot API. Null is returned if the member is not found.
+     * 
+     * @return object|null
+     */
+    public function forumMember()
```

Comment:
Let's keep this logic in the sync job itself rather than tying it to the member model. There should be a clear separation between things that are tied directly to the model and things that actively make external requests. We also probably won't have a need for this call anywhere else except for in the context of the sync.